src/main/java/com/happeningnow/happeningnow/HelloWorldController.java

```diff
@@ -0,0 +1,12 @@
+package com.happeningnow.happeningnow;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+    @RequestMapping("/")
+    public String Application() {
```

Comment:
The method name does not follow the Java naming convention. Please consider using camelCase, and use a more appropriate name, like `home` or `getHelloWorldMessage`

Reply:
ok